### PR TITLE
[8] Implement UI of uploading CSV file

### LINKED ISSUE #8: [UI] As a user, I can upload a keywords-contained CSV file which will be stored
## Why

User needs an upload zone to request searching keywords

## Who Benefits?

The user who needed to use the searching keyword by keyword from uploaded CSV

### PR DESCRIPTION
**Pull request by nvminhtue**
close #8 

## What happened

The home screen should have a CSV uploading input
 
## Insight

- Implement UI for uploading zone
- Implement feature test
 
## Proof Of Work

image

### CODE REVIEW

File: lib/elixir_internal_certificate_web/templates/user_search/index.html.heex [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<section class="phx-hero">
+  <.form id="keyword-upload-form" let={f} for={@conn} action={Routes.user_search_path(@conn, :index)} multipart={true}>
+    <%= label f, :File, "Select CSV file, maximum 100 keywords contained", class: "form-label" %>
```
Comment (andyduong1920) [inline lib/elixir_internal_certificate_web/templates/user_search/index.html.heex]: maximum `1000` keywords contained

File: test/elixir_internal_certificate_web/features/home_page/upload_csv_test.exs [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      |> assert_has(Query.css("#keyword-upload-form_file"))
+  )
+
+  feature "upload file", %{session: session} do
+    upload_field = Query.file_field("file")
+
+    session
+    |> login_user()
+    |> attach_file(upload_field, path: "test/support/assets/files/upload.csv")
+    |> take_screenshot()
```
Comment (andyduong1920) [inline test/elixir_internal_certificate_web/features/home_page/upload_csv_test.exs]: do we need this action

Comment (andyduong1920) [inline test/elixir_internal_certificate_web/features/home_page/upload_csv_test.exs]: do we need this action

File: test/elixir_internal_certificate_web/features/home_page/upload_csv_test.exs [hunk 3]
```diff
@@ -0,0 +1,31 @@
+defmodule ElixirInternalCertificateWeb.HomePage.UploadCSVTest do
+  use ElixirInternalCertificateWeb.FeatureCase, async: true
+
+  feature("uploading page", %{session: session},
```
Comment (andyduong1920) [inline test/elixir_internal_certificate_web/features/home_page/upload_csv_test.exs:4]: How about changing the `uploading page` to `View the upload file form`

Comment (andyduong1920) [inline test/elixir_internal_certificate_web/features/home_page/upload_csv_test.exs:4]: How about changing the `uploading page` to `View the upload file form`

**Review by andyduong1920 — COMMENTED** [review]
Some small comments @nvminhtue

**Review by andyduong1920 — APPROVED** [review]

**Review by junan — APPROVED** [review]

**Comment by andyduong1920** [comment]
@nvminhtue we can merge this PR as it received 2 approvals already 👍

**Pull request merged**